src/main/java/org/opensearch/neuralsearch/processor/chunker/Chunker.java

```diff
@@ -14,6 +14,7 @@
 public interface Chunker {
 
     String MAX_CHUNK_LIMIT_FIELD = "max_chunk_limit";
+    String STRING_TOBE_CHUNKED_FIELD = "string_tobe_chunked_count";
```

Comment:
Rename this and make this static final

Reply:
Modifier stack and final is redundant for interface